docs/pbc/all/tax-management/202410.0/base-shop/third-party-integrations/vertex/install-vertex/integrate-vertex-validator.md

```diff
@@ -1,22 +1,77 @@
 ---
 title: Integrate Vertex Validator
 description: Find out how you can integrate the Vertex Validator into your Spryker shop
-last_updated: Jan 8, 2025
+last_updated: Jun 4, 2025
 template: howto-guide-template
 redirect_from:
   - /docs/pbc/all/tax-management/202410.0/base-shop/third-party-integrations/vertex/install-vertex/integrate-taxamo.html
 ---
 
-To integrate Vertex Validator, follow the steps:
+To integrate Vertex Validator, follow these steps.
 
-1. Update [spryker/tax-app](https://github.com/spryker/tax-app) to `0.4.0` and install the [spryker/tax-app-rest-api](https://github.com/spryker/tax-app-rest-api) module:
+## 1. Install required modules
+
+Update [spryker/tax-app](https://github.com/spryker/tax-app) to `^0.4.2`, update [spryker/glossary-storage](https://github.com/spryker/glossary-storage) to `^1.11.2`, and install the [spryker/tax-app-rest-api](https://github.com/spryker/tax-app-rest-api) module:
+
+```bash
+composer require spryker/tax-app:"^0.4.2" spryker/tax-app-rest-api:"^0.2.0" spryker/glossary-storage:"^1.11.2" --update-with-dependencies
+```
```

Comment:
because glossary-storage is already dependency of [tax-app-rest-api](https://github.com/spryker/tax-app-rest-api/blob/master/composer.json#L8)

```suggestion
Update [spryker/tax-app](https://github.com/spryker/tax-app) to `^0.4.2` and install the [spryker/tax-app-rest-api](https://github.com/spryker/tax-app-rest-api) module:

```bash
composer require spryker/tax-app:"^0.4.2" spryker/tax-app-rest-api:"^0.2.0" --update-with-dependencies
```
```